.storybook/main.ts

```diff
@@ -70,4 +73,47 @@ module.exports = {
       plugins: [],
     };
   },
+  webpackFinal(config, { configType }) {
+    if (configType === 'DEVELOPMENT') {
+      updateCSSLoaderPlugin(config);
+    }
+    return config;
+  },
 };
+
+/**
+ * Updates the `css-loader` webpack plugin to make class names human readable.
+ *
+ * NOTE: This should only be used for local development.
+ */
+function updateCSSLoaderPlugin(config: Configuration): Configuration {
+  config.module?.rules?.forEach((rule) => {
+    if (rule && typeof rule === 'object' && Array.isArray(rule.use)) {
```

Comment:
An alternative could be to use webpack-merge's [mergeWithRules](https://github.com/survivejs/webpack-merge#mergewithrules).

Not sure if that'll make this simpler or not, but could be worth trying out.

Reply:
I'm going to leave this as is for now. I tried it out but merging is kind of weird because the value of `loader` provided from the Storybook webpack config is an absolute path to the `css-loader` library file. 

Specifically,

```ts
const ourConfig: Configuration = {
    module: {
      rules: [
        {
          test: /\.css$/,
          use: [
            {
              loader: 'css-loader',
              options: {
                modules: {
                  localIdentName: '[name]__[local]--[hash:base64:5]',
                },
              },
            },
          ],
        },
      ],
    },
  };

  return mergeWithRules({
    module: {
      rules: {
        test: 'match',
        use: {
          loader: 'match',
          options: 'merge',
        },
      },
    },
  })(theirConfig, ourConfig);
```

does not merge the `css-loader` correctly 
![image](https://github.com/chanzuckerberg/edu-design-system/assets/94476616/ea9d535b-7c8a-4265-bf1f-df14d1a07c03)

Hardcoding the path to `node_modules/css-loader/dist/cjs.js` felt a bit weird so decided to not do it